domain/domain.go

```diff
@@ -140,7 +141,13 @@ func (do *Domain) loadInfoSchema(handle *infoschema.Handle, usedSchemaVersion in
 		return 0, nil, fullLoad, err
 	}
 
-	newISBuilder, err := infoschema.NewBuilder(handle).InitWithDBInfos(schemas, neededSchemaVersion)
+	bundles, err := infosync.GetAllRuleBundles(nil)
+	// ignore if placement rules feature is not enabled
+	if err != nil && !strings.HasPrefix(err.Error(), `"placement rules feature is disabled"`) {
```

Comment:
Yes, in case of PD disabled placement rules feature.
It is not handled smoothly in the past. First, it is causing troubles on tests. Second, it will be reported as something like `can not unmarshal string into xxx`.
The root cause is that `errors.Wrapf` in `doRequest` will wrap a nil error, the error returned by reading `res.Body`..

Reply:
I am considering using a common error for ignoring errors.

Reply:
done